pytorch_to_returnn/naming/naming.py

```diff
@@ -325,8 +325,10 @@ def register_input(self, tensor: _types.Tensor, returnn_data: Data) -> Data:
       feature_dim_axis=returnn_data.feature_dim_axis_or_unspecified,
       available_for_inference=True)
     entry.returnn_axis_from_torch_axis = {i: i for i in range(returnn_data.batch_ndim)}
-    if entry.returnn_data.have_batch_axis():
-      tensor.shape[entry.returnn_data.batch_dim_axis].is_batch_dim = True
+    for axis in range(returnn_data.batch_ndim):
+      tensor.shape[axis].dim_tag = entry.returnn_data.dim_tags[axis]
+      tensor.shape[axis].tensor = tensor
```

Comment:
We can also pass the `tensor` directly in `SizeValue.__new__` (in `Tensor.__init__`), right?